browser/css/notebookbar.css

```diff
@@ -81,8 +81,8 @@
 }
 
 .notebookbar-shortcuts-bar .notebookbar.unotoolbutton {
-	height: 27px;
```

Comment:
but in here it seems we are changing from 27 to 24 . Does that affect or cause regressions? I have tested and it seems that is save